examples/util/src/lib.rs

```diff
@@ -1,11 +1,20 @@
 use rand::{thread_rng, Rng};
-use std::process::{Command, Stdio};
+use std::process::{Child, Command, Stdio};
 use std::{io::ErrorKind, time::Duration};
 use tokio::time::sleep;
 
 const PROMETHEUS_CONFIG_PATH: &str = concat!(env!("CARGO_MANIFEST_DIR"), "/prometheus.yml");
 
-pub fn run_prometheus() {
+pub struct ChildGuard(Child);
+
+impl Drop for ChildGuard {
+    fn drop(&mut self) {
+        self.0.kill().expect("Failed to kill prometheus");
```

Comment:
`panic!`ing on `drop` implementations is really nasty (will most likely abort) but that seems fine in this case

Reply:
Should we just swallow the error instead?

Reply:
we shoudl just handler the error, e.g print out the error that was returned and probably also the pid of the child process so the user can investigate it easier